routing/router.go

```diff
@@ -922,6 +922,9 @@ type LightningPayment struct {
 	// the first hop.
 	PaymentHash [32]byte
 
+	// The array of routes which might be set by the sender to execute
```

Comment:
Rather than taking on the route field to this struct (which only describes the payment details), the `SendToRoute` method should be modified to take two params: the full route, and the payment details.